games-rpg/eternal-lands/files/el-wrapper

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
```

Comment:
This should be `/bin/bash` because you've used Bashisms in this script (`[[`) and perhaps you should add `-e` so that you don't need `|| exit` all over the place.

Reply:
Right. My bad.